Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Container image that runs your code
```

Comment:
This `Dockerfile` should not be here - we will use mender-ci-tools image

Reply:
Looks like some leftovers from https://github.com/TheYoctoJester/mender-push-artifact

It's actually a question if we need to keep all commits history or not.